docs/docs/deployment/hosting/kubernetes.md

```diff
@@ -16,7 +16,24 @@ helm install -n flagsmith --create-namespace flagsmith flagsmith/flagsmith
 kubectl -n flagsmith port-forward svc/flagsmith-frontend 8080:8080
 ```
 
-Then view `http://localhost:8080` in a browser. This will install using default options, in a new namespace `flagsmith`.
+Then view `http://localhost:8080` in a browser. This will install the chart using default options, in a new namespace
+`flagsmith`.
+
+If you want to make changes to the default setup you can make a local copy of the
```

Comment:
They _can_, but they don't _have to_. I'll very much prefer to emphasize that the default values file should be used for reference, not for input.

I wonder if we offered a [json schema](https://www.arthurkoziel.com/validate-helm-chart-values-with-json-schemas/) we could avoid this bit of docs altogether.

Reply:
Can you provide a suggestion?

Reply:
See https://github.com/Flagsmith/flagsmith/pull/2222/files#r1222919381